internal/convert/registryv1.go

```diff
@@ -274,15 +271,18 @@ func Convert(in RegistryV1, installNamespace string, targetNamespaces []string)
 		permissions = nil
 	}
 
-	for _, permission := range permissions {
-		saName := saNameOrDefault(permission.ServiceAccountName)
-		name, err := generateName(fmt.Sprintf("%s-%s", in.CSV.Name, saName), permission)
-		if err != nil {
-			return nil, err
+	for _, ns := range targetNamespaces {
```

Comment:
What should we do if this list is empty?

Reply:
We already have validation for that in 2 places:
1. We ensure if targetNamespaces is empty, AllNs is supported: https://github.com/operator-framework/rukpak/blob/b4e9976e748299623d5e2548c7bf1b3a0c40edf1/internal/convert/registryv1.go#L147
2. If CSV supports AllNs or OwnNs, then accordingly fill it in here: https://github.com/operator-framework/rukpak/blob/b4e9976e748299623d5e2548c7bf1b3a0c40edf1/internal/convert/registryv1.go#L198
3. If its AllNs mode, then we elevate permissions to scaffold only CR and CRD and set permissions to nil: https://github.com/operator-framework/rukpak/blob/b4e9976e748299623d5e2548c7bf1b3a0c40edf1/internal/convert/registryv1.go#L274.

Reply:
Thanks!